hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OzoneFSUtils.java

```diff
@@ -317,9 +317,6 @@ public static boolean canEnableHsync(ConfigurationSource conf, boolean isClient)
     if (confHBaseEnhancementsAllowed) {
       return confHsyncEnabled;
     } else {
-      LOG.warn("Ignoring {} = {} because HBase enhancements are disallowed. To enable it, set {} = true as well.",
-          OzoneConfigKeys.OZONE_FS_HSYNC_ENABLED, confHsyncEnabled,
-          confKey);
       return false;
```

Comment:
ah I see the mistake I made in the previous jira. Let's do this instead. This was there for a reason as I mentioned above. But I missed the extra check.

```suggestion
      if (confHsyncEnabled) {
	      LOG.warn("Ignoring {} = {} because HBase enhancements are disallowed. To enable it, set {} = true as well.",
	          OzoneConfigKeys.OZONE_FS_HSYNC_ENABLED, confHsyncEnabled,
	          confKey);
      }
      return false;
```

Reply:
The check can also become:

```java
    if (confHBaseEnhancementsAllowed) {
      return confHsyncEnabled;
    } else if (confHsyncEnabled) {
      LOG.warn("Ignoring {} = {} because HBase enhancements are disallowed. To enable it, set {} = true as well.",
          OzoneConfigKeys.OZONE_FS_HSYNC_ENABLED, confHsyncEnabled,
          confKey);
    }
    return false;
```

Reply:
Done